src/main.module.ts

```diff
@@ -12,6 +12,7 @@ import { LoggerModule } from 'nestjs-pino';
     }),
     LoggerModule.forRoot({
       pinoHttp: {
+        // TODO APIM-97: it would be good to have a way to configure the log-level
```

Comment:
I'll raise a ticket for this (I thought of it here because our logging is fixed at INFO and above currently, and I've added DEBUG level messages in this PR)

Reply:
Agree this should be controlled by an environment variable.

Reply:
Raised APIM-184